src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  color: $--main-color-text;
+  font-weight: 400;
```

Comment:
This is default value

Reply:
https://github.com/mate-academy/layout_catalog/pull/3844/files#r1398232415
not fixed